Course_Part_04/src/Assignment13.java

```diff
@@ -39,6 +39,7 @@ public static void main(String[] args) {
 
     private static double calculator(double a, double b, String operator) {
```

Comment:
keurige methode en nette switch waar in je meteen returned, super! De return in de default is een beetje dubieus omdat een ongeldige operator nu niet opgemerkt wordt. Je zou een fout kunnen opnemen met een melding 'Unsupported operator' of iets dergelijks.